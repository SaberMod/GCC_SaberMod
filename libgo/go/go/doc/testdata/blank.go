// Copyright 2014 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Package blank is a go/doc test for the handling of _.
// See issue 5397.
package blank

type T int

// T constants.
const (
	_ T = iota
	T1
	T2
)

// Package constants.
const (
	_ int = iota
	I1
	I2
)

// Blanks not in doc output:

// S has a padding field.
type S struct {
	H uint32
	_ uint8
	A uint8
}

func _() {}

type _ T

var _ = T(55)
