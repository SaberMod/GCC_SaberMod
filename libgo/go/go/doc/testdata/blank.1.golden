// Package blank is a go/doc test for the handling of _. See issue ...
PACKAGE blank

IMPORTPATH
	testdata/blank

FILENAMES
	testdata/blank.go

CONSTANTS
	// Package constants. 
	const (
		_	int	= iota
		I1
		I2
	)


VARIABLES
	// 
	var _ = T(55)


FUNCTIONS
	// 
	func _()


TYPES
	// S has a padding field. 
	type S struct {
		H	uint32
		_	uint8
		A	uint8
	}

	// 
	type T int

	// T constants. 
	const (
		_	T	= iota
		T1
		T2
	)

